src/components/modals/Send/fields/AmountField.js

```diff
@@ -39,10 +39,10 @@ class AmountField extends Component<
   componentDidMount() {
     this.resync()
   }
-  componentDidUpdate(nextProps: *) {
+  componentDidUpdate(prevProps: *) {
```

Comment:
indeed, good catch

Reply:
It actually messed with me during debugging, I was seeing the previous props and thinking "wait, that's not right..." :thinking: :smile: